tools/killsnoop.py

```diff
@@ -83,7 +83,7 @@
 {
     struct data_t data = {};
     struct val_t *valp;
-    u32 pid = bpf_get_current_pid_tgid();
+    u32 pid = bpf_get_current_pid_tgid() >> 32;
```

Comment:
again, pid is used as the key here. we should use tid as the key, pid to the user space.